<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QMainWindow" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>388</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Сервер</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="x_label">
      <property name="text">
       <string>Координата X:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="x_edit">
      <property name="toolTip">
       <string>Допустимое значение: от 0 до 63</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>от 0 до 63</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="y_label">
      <property name="text">
       <string>Координата Y:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="y_edit">
      <property name="toolTip">
       <string>Допустимое значение: от -32 до 31</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>от -32 до 31</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="velocity_label">
      <property name="text">
       <string>Скорость:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="velocity_edit">
      <property name="toolTip">
       <string>Допустимое значение: от 0 до 255</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>от 0 до 255</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="mode_label">
      <property name="text">
       <string>Режим работы:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="mode_edit">
      <property name="toolTip">
       <string>Допустимое значение: от 0 до 3</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>от 0 до 3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="state_label">
      <property name="text">
       <string>Состояние:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="state_edit">
      <property name="toolTip">
       <string>Допустимое значение: от 0 до 3</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>от 0 до 3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="acc_label">
      <property name="text">
       <string>Ускорение:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="acc_edit">
      <property name="toolTip">
       <string>Допустимое значение: от -12.7 до 12.8</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>от -12.7 до 12.8</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="power_label">
      <property name="text">
       <string>Мощность:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="power_edit">
      <property name="toolTip">
       <string>Допустимое значение: от 0 до 130</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>от 0 до 130</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="5">
     <widget class="QLabel" name="reserve_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Резерв (побитово)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="5">
     <layout class="QGridLayout" name="bits_grdlayout">
      <item row="1" column="13">
       <widget class="QSpinBox" name="bit2_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QSpinBox" name="bit7_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="bit15_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="bit14_label">
        <property name="text">
         <string>14</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="13">
       <widget class="QLabel" name="bit2_label">
        <property name="text">
         <string>2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="14">
       <widget class="QSpinBox" name="bit1_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="bit13_label">
        <property name="text">
         <string>13</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="bit8_label">
        <property name="text">
         <string>8</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="bit7_label">
        <property name="text">
         <string>7</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="15">
       <widget class="QLabel" name="bit0_label">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QLabel" name="bit4_label">
        <property name="text">
         <string>4</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="bit12_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="12">
       <widget class="QSpinBox" name="bit3_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="bit6_label">
        <property name="text">
         <string>6</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="10">
       <widget class="QSpinBox" name="bit5_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="11">
       <widget class="QSpinBox" name="bit4_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="bit13_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QSpinBox" name="bit8_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QSpinBox" name="bit10_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="bit9_label">
        <property name="text">
         <string>9</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="bit12_label">
        <property name="text">
         <string>12</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QLabel" name="bit5_label">
        <property name="text">
         <string>5</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QSpinBox" name="bit9_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="15">
       <widget class="QSpinBox" name="bit0_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="14">
       <widget class="QLabel" name="bit1_label">
        <property name="text">
         <string>1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="bit10_label">
        <property name="text">
         <string>10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QLabel" name="bit3_label">
        <property name="text">
         <string>3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="bit15_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>15</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="bit14_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="bit11_label">
        <property name="text">
         <string>11</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="bit11_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QSpinBox" name="bit6_spbx">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0" colspan="5">
     <widget class="Line" name="line">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="ip_label">
      <property name="text">
       <string>IP-адрес клиента:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2" colspan="3">
     <layout class="QHBoxLayout" name="ip_hrzlayout">
      <item>
       <widget class="QLineEdit" name="ip1_ledit">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>127</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ipdot1_lbl">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ip2_ledit">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ipdot2_lbl">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ip3_ledit">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ipdot3_lbl">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ip4_ledit">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="port_label">
      <property name="text">
       <string>Порт:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLineEdit" name="port_ledit">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>2424</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="update_button">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="5">
     <widget class="QLabel" name="warning_lbl">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>x_edit</tabstop>
  <tabstop>y_edit</tabstop>
  <tabstop>velocity_edit</tabstop>
  <tabstop>mode_edit</tabstop>
  <tabstop>state_edit</tabstop>
  <tabstop>acc_edit</tabstop>
  <tabstop>power_edit</tabstop>
  <tabstop>bit15_spbx</tabstop>
  <tabstop>bit14_spbx</tabstop>
  <tabstop>bit13_spbx</tabstop>
  <tabstop>bit12_spbx</tabstop>
  <tabstop>bit11_spbx</tabstop>
  <tabstop>bit10_spbx</tabstop>
  <tabstop>bit9_spbx</tabstop>
  <tabstop>bit8_spbx</tabstop>
  <tabstop>bit7_spbx</tabstop>
  <tabstop>bit6_spbx</tabstop>
  <tabstop>bit5_spbx</tabstop>
  <tabstop>bit4_spbx</tabstop>
  <tabstop>bit3_spbx</tabstop>
  <tabstop>bit2_spbx</tabstop>
  <tabstop>bit1_spbx</tabstop>
  <tabstop>bit0_spbx</tabstop>
  <tabstop>ip1_ledit</tabstop>
  <tabstop>ip2_ledit</tabstop>
  <tabstop>ip3_ledit</tabstop>
  <tabstop>ip4_ledit</tabstop>
  <tabstop>port_ledit</tabstop>
  <tabstop>update_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
